libs/core/src/lib/carousel/carousel.component.ts

```diff
@@ -557,8 +546,10 @@ export class CarouselComponent implements OnInit, AfterContentInit, AfterViewIni
     private _subscribeToRtl(): void {
         const refreshDirection = () => {
             this.dir = this._rtlService?.rtl.getValue() ? 'rtl' : 'ltr';
+            this._carouselService.isRtl = this.dir === 'rtl';
```

Comment:
`this.isRtl()` can be used for simplification.

Reply:
fixed